ru/turing/courses/lesson2/animals/Animal.java

```diff
@@ -1,4 +1,9 @@
 package ru.turing.courses.lesson2.animals;
 
-public class Animal {
+public abstract class Animal {
+public void sleep(){
+    System.out.println("Zzzz");
+}
+public abstract void eat();
+public abstract int Kol_legs();
```

Comment:
```suggestion
public abstract int amountOfLegs();
```
В джаве принято писать в соответствии с CamelCase